kibble/__main__.py

```diff
@@ -15,9 +15,91 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import click
+from getpass import getpass
+
+from kibble.cli import setup_command
+from kibble.version import version as kv
+
+from kibble.configuration import conf
+
+
+def get_user_input(msg: str, secure: bool = False):
+    value = None
+    while not value:
+        value = getpass(msg) if secure else input(msg)
+    return value
+
+
+@click.group()
+def cli():
+    """A simple command line tool for kibble"""
+
+
+@cli.command("version", short_help="displays the current kibble version")
+def version():
+    click.echo(kv)
+
+
+@cli.command("setup", short_help="starts the setup process for kibble")
+@click.option(
+    "-u",
+    "--uri",
+    default=conf.get("elasticsearch", "conn_uri"),
+    help="Pre-defined connection uri for ElasticSearch.",
```

Comment:
```suggestion
    help="connection uri for ElasticSearch.",
```
Let's remove this "pre-defined", in my opinion it looks weird in the `kibble setup --help`